libs/SoloPlay/Tools/Overlay.js

```diff
@@ -29,6 +29,7 @@ const Overlay = {
 			let currInGame = getTickCount() - me.gamestarttime;
 			let totalTime = Developer.formatTime(this.GameTracker.Total + currInGame);
 			let totalInGame = Developer.formatTime(this.GameTracker.InGame + currInGame);
+			let totalDays = Developer.totalDays(this.GameTracker.Days);
```

Comment:
`let totalDays = Developer.totalDays(this.GameTracker.Total + currInGame);`

just pull the info from total rather than create a whole new variable that's storing the exact same information